vocdoni/bridge-ui
# Feat: Token fetching improvement

# Short description
The application right now is fetching the tokens and not storing them in a cache, making a bad UX when a user gets in the app, also the way we handle the request has been improved with [SWR](https://swr.vercel.app/), which allow us to improve the way we handle the errors and data of every request.

# How can it be tested
- Go to the app and go to `/tokens` route, then refresh and see that all the token cards appears
- Go to staging (https://bridge-ui-staging.herokuapp.com/) and do the same
- See the difference

# Tracker ID
https://linear.app/aragon/issue/VOT-107/load-tokens-service-improvements
